src/commands/help.ts

```diff
@@ -0,0 +1,68 @@
+import { jaroWinkler } from '@skyra/jaro-winkler';
+import chalk from 'chalk';
+
+import { ApifyCommand, commandRegistry } from '../lib/command-framework/apify-command.js';
+import { Args } from '../lib/command-framework/args.js';
+import { renderHelpForCommand, renderMainHelpMenu } from '../lib/command-framework/help.js';
+import { error } from '../lib/outputs.js';
+
+export class HelpCommand extends ApifyCommand<typeof HelpCommand> {
+	static override name = 'help' as const;
+
+	static override description = 'Prints out help about a command, or all available commands.';
+
+	static override hidden = true;
+
+	static override args = {
+		commandString: Args.string({
+			required: false,
+			description: 'The command to get help for.',
+			catchAll: true,
+		}),
+	};
+
+	async run() {
+		const { commandString } = this.args;
+
+		if (!commandString || commandString.toLowerCase().startsWith('help')) {
+			const helpMenu = renderMainHelpMenu(this.entrypoint);
+
+			console.log(helpMenu);
+
+			return;
+		}
+
+		const lowercasedCommandString = commandString.toLowerCase();
+
+		const command = commandRegistry.get(lowercasedCommandString);
+
+		if (!command) {
+			const allCommands = [...commandRegistry.keys()];
+
+			const closestMatches = allCommands.filter((cmd) => {
+				const lowercased = cmd.toLowerCase();
+
+				return jaroWinkler(lowercasedCommandString, lowercased) >= 0.95;
```

Comment:
Jaro-Winkler similarity is quite unstable for shorter strings (`help -> hlp` = 92.50, meaning no match). Can we e.g. combine this and raw edit distance (if match e.g. levenshtein <= 3 or jaroWinkler >= 0.95)?

Reply:
Additionally (out of scope of this PR), can we also have this for regular command invocation? :eyes: (e.g. `apify hlp` -> `Did you mean: help`)

Reply:
Yes and yes! Good callouts, will PR them :D (and unify them in one function